buildkitd/buildkitd.go

```diff
@@ -84,7 +84,11 @@ func NewClient(ctx context.Context, console conslogging.ConsoleLogger, image, co
 		remoteConsole := console
 		if settings.SatelliteName != "" {
 			remoteConsole = console.WithPrefix("satellite")
-			remoteConsole.Printf("Connecting to %s...", settings.SatelliteDisplayName)
+			selfHostedInfo := ""
+			if settings.SatelliteIsSelfHosted {
+				selfHostedInfo = fmt.Sprintf(" (self-hosted at %s)", settings.BuildkitAddress)
+			}
+			remoteConsole.Printf("Connecting to %s...%s", settings.SatelliteDisplayName, selfHostedInfo)
```

Comment:
having `...` in the middle seems a bit odd in the case of self-hosted; I would just use an `else` in this case, e.g.

```
			if settings.SatelliteIsSelfHosted {
				remoteConsole.Printf("Connecting to %s (hosted at %s)...", settings.SatelliteDisplayName, selfHostedInfo)
			} else {
				remoteConsole.Printf("Connecting to %s...", settings.SatelliteDisplayName)
			}
```

or use a `strings.Builder` if you need greater control.